bin/fts_quickstart

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+echo "Installing Java SDK..."
+# Find our package manager
+if VERB="$( command -v apt-get )" 2> /dev/null; then
+   echo "Debian-based OS detected"
+   sudo apt-get install -y openjdk-8-jdk 2> /dev/null
+elif VERB="$( command -v yum )" 2> /dev/null; then
+   echo "Modern Red Hat-based OS detected"
+   sudo yum install -y java-1.8.0-openjdk.x86_64 2> /dev/null
+else
+   echo "No valid package manager detected; try one of apt-get, yum."
+   exit 1
+fi
+
+if [ ! -d "/opt/solr" ]; then
+    printf "Installing Solr 4"
+    # Acquire and unzip solr4
+    wget http://archive.apache.org/dist/lucene/solr/4.10.4/solr-4.10.4.tgz 2> /dev/null && printf "."
+    tar -xf solr-4.10.4.tgz 2> /dev/null && printf "."
+    rm solr-4.10.4.tgz 2> /dev/null && printf "."
+
+    # Set the defaults in /opt/solr
+    sudo mv solr-4.10.4 /opt/solr 2> /dev/null && printf "."
+    mv /opt/solr/example /opt/solr/core 2> /dev/null && echo "."
+fi
+
+if [ ! -f "/etc/init.d/solr" ]; then
+    echo "Installing Solr daemon..."
+    # Set up the daemon so that solr will run on startup
+    sudo cp vendor/silverstripe/fulltextsearch/docs/examples/daemon_script /etc/init.d/solr 2> /dev/null
+    sudo chmod +x /etc/init.d/solr 2> /dev/null
+    sudo chkconfig --add solr 2> /dev/null
+fi
+
+# Get solr running
+sudo /etc/init.d/solr start 2> /dev/null
+
+# Determine application dir
+if [ -d app ]; then
+    APPDIR="app"
+elif [ -d mysite ]; then
+    APPDIR="mysite"
+else
+    echo "Can't detect application dir - skipping default index creating"
+    exit 1
+fi
+
+# Check to see if it has been configured in _config.php
+grep -i "Solr::configure_server(" "$APPDIR/_config.php" 2> /dev/null
```

Comment:
you can use the `-q` argument with grep to suppress output and only give you an exit code